src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -107,6 +109,11 @@
 ])
 
 
+class Pollination(Enum):
+  RANDOM = 1
```

Comment:
It may still be useful for these values to be strings since it will be easier to understand when looking at the bigquery stats.

Reply:
Done